frontend/src/static/js/api.js

```diff
@@ -58,6 +58,7 @@ window.api = {
             displayName: commits.authorDisplayNameMap[author],
             dailyCommits: commits.authorDailyContributionsMap[author],
             totalCommits: commits.authorFinalContributionMap[author],
+            fileTypeContribution: commits.authorFileTypeContributionMap[author],
```

Comment:
We are using the term 'file formats' throughout RepoSense, so do refactor the variable and method names to 'fileFormat' as well accordingly (including the backend)